templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/service/impl/config/BaseWebSecurityConfig.java

```diff
@@ -75,8 +75,8 @@ public void configure(HttpSecurity http) throws Exception {
         // define all urls that are not to be secured
         .authorizeRequests().antMatchers(unsecuredResources).permitAll().anyRequest().authenticated().and()
 
-        // activate crsf check for a selection of urls (but not for login & logout)
-        .csrf().requireCsrfProtectionMatcher(new CsrfRequestMatcher()).and()
+        //disabling CSRF
+        .csrf().disable()
```

Comment:
how does out spring-boot-starter enable it back?
Did you consider a property or to detect somehow if the starter is present?